lib/util.js

```diff
@@ -186,8 +186,10 @@ const util = {
 
     if (process.platform === 'win32') util.updateOmahaMidlFiles()
 
-    const args = util.buildArgsToString(config.buildArgs())
-    util.run('gn', ['gen', config.outputDir, '--args="' + args + '"'], options)
+    if (!options.skip_gn_gen) {
```

Comment:
Can we just determine this automatically instead?  Maybe keeping old and new hash of last used config?

Reply:
If we only consider `args.gn`, it would be possible.
However, we should re run `gn gen` when we touch anything of gn/gni files.